yjit/bindgen/src/main.rs

```diff
@@ -0,0 +1,286 @@
+//! See https://docs.rs/bindgen/0.59.2/bindgen/struct.Builder.html
+//! This is the binding generation tool that the YJIT cruby module talks about.
+//! More docs later once we have more experience with this, for now, check
+//! the output to make sure it looks reasonable and allowlist things you want
+//! to use in Rust.
+extern crate bindgen;
```

Comment:
nit: Using `extern crate` is no longer necessary since the 2018 edition in the majority of the cases, including this case.